docs/product-user-guide/src/main/asciidoc/drl-rules-WHEN-proc.adoc

```diff
@@ -0,0 +1,160 @@
+[id='drl-rules-WHEN-proc']
+= Adding WHEN conditions in DRL rules
+
+The `when` part of the rule contains the conditions that must be met to execute an action. For example, if a bank requires loan applicants to have over 21 years of age, then the `when` condition of an "Underage" rule would be `Applicant( age < 21 )`. Conditions consist of a series of stated patterns and constraints, with optional bindings and other supported Java elements, based on the available data objects in the package.
+
+.Prerequisites
+* The `package` is defined at the top of the DRL file. This should have been done for you when you created the file.
+* The `import` list of data objects used in the rule is defined below the `package` line of the DRL file. Data objects can be from this package or from another package in {CENTRAL}.
+* The `rule` name is defined in the format `rule "name"` below the `package`, `import`, and other lines that apply to the entire DRL file. The same rule name cannot be used more than once in the same package. Optional rule attributes (such as `salience` or `no-loop`) that define rule behavior are below the rule name, before the `when` section.
+
+.Procedure
+. In the DRL designer, enter `when` within the rule to begin adding condition statements. The `when` section consists of zero or more fact patterns that define conditions for the rule.
++
+If the `when` section is empty, then actions in the `then` section are executed once every time the decision engine creates a new session. This is useful when certain actions apply to all rules every session, regardless of conditions. *<@Michael (7 of 13): I tried to make this clearer, but not sure I'm right. This close? And again, is "decision engine" interchangeable with "working memory" of the decision engine, just for the sake of simplicity?>*
++
+.Rule without conditions
+[source,java]
+----
+rule "Approve all"
+  when   // empty
+
+  then   // actions to be executed once
+    application.setApproved( true );
+    application.setExplanation( "Approve all applicants" );
+end
+
+// The above rule is internally rewritten as:
+
+rule "Approve all"
+  when
+    eval( true )
+  then
+    application.setApproved( true );
+    application.setExplanation( "Approve all applicants" );
+end
+----
++
+. Enter a pattern for the first condition to be met, with optional constraints, bindings, and other supported Java elements. A basic pattern format is `patternBinding : patternType ( constraints )`. Patterns are based on the available data objects in the package and define the conditions to be met in order to trigger actions in the `then` section.
```

Comment:
@sterobin It's not "other supported **Java** elements" but "other supported **DRL** elements".